pkg/extensions/extension.go

```diff
@@ -16,7 +19,8 @@ type Extension interface {
 //go:generate moq -rm -out manager_mock.go . ExtensionManager
 type ExtensionManager interface {
 	List(includeMetadata bool) []Extension
-	Install(url string, stdout, stderr io.Writer) error
+	InstallGit(url string, stdout, stderr io.Writer) error
+	InstallBin(client *http.Client, repo ghrepo.Interface) error
```

Comment:
Do InstallGit and InstallBit have to be published? Couldn't they just stay an implementation detail of `Install`?

Reply:
I wasn't sure. Their signatures and implementations are so different it felt weird to combine them, but I'm certainly not opposed to that.